src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -36,15 +36,12 @@ public void testIsFull() {
 			Assert.assertEquals(false, result);
 			ryhmer.countIn(888);
 		}
-
-		boolean result = ryhmer.isFull();
-		Assert.assertEquals(true, result);
```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project. STACK_CAPACITY is 12 so 12 elements should be added to the stack.